Resources/Prototypes/_Sunrise/Entities/Objects/Weapons/Guns/Pistols/pistols.yml

```
@@ -47,18 +100,79 @@
+  parent: BaseWeaponPistolSunrise
   id: WeaponPistolG22
-  suffix: Lethal
+  suffix: Sunrise, Security
   components:
   - type: Sprite
     sprite: _Sunrise/Objects/Weapons/Guns/Pistols/glock22/big.rsi
   - type: Item
     sprite: _Sunrise/Objects/Weapons/Guns/Pistols/glock22/tiny.rsi
+  - type: Tag
+    tags:
+    - WeaponPistolG22
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)